### PR TITLE
Regex style guideline to use Slashy Strings

### PR DESCRIPTION
**Pull request by brunobowden**
- Also add newlines in markdown to aid display with Android Studio markdown plugin

### CODE REVIEW

**Comment by brunobowden** [comment]
@confile, @advayDev1 - Regex style guideline to use Slashy Strings

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -112,18 +115,27 @@ String message = "the count is ${count}"         // WRONG
...
+
 
 ### Testing your code
-Unit tests must be written for all new code and major changes. Please follow the example
-of existing tests within the code.
+Unit tests must be written for all new code and major changes. Please follow the examples
+of existing tests.
 
-To run the build and all the tests:
+Running the build also runs all the unit tests:
```
Comment (advayDev1) [inline CONTRIBUTING.md]: "Running `build` also runs..."

**Comment by advayDev1** [comment]
lgtm

**Comment by brunobowden** [comment]
"Running `build`..." fixed done and also merged.

**Pull request merged**